src/main/java/objects/battle/enemy/factory/EnemyFactory.java

```diff
@@ -57,33 +58,41 @@ public Enemy createEnemy(String name) throws Exception {
      * @param enemyInfo EnemeyInfo of the name given
      * @return gimmick that the enemy has (from the database)
      */
-    public Gimmick translateGimmick(String name, EnemyInfo enemyInfo)
-            throws GimmickNotFoundException {
+    public Gimmick translateGimmick(String name, EnemyInfo enemyInfo) {
         switch (name) {
             case "health":{
                 GimmickData healthGimmickData = this.gimmickDataManager.fetchGimmickData(name);
                 int triggerHealth = Integer.parseInt(healthGimmickData.trigger);
-                return new HealthGimmick(enemyInfo, triggerHealth);
+                return new Gimmick(GimmickType.HEALTH, enemyInfo, triggerHealth, 0, 1, null);
```

Comment:
recommend to create a gimmickBuilder to eliminate the need for multiple null/default constructor values